From 4535b885de656d0d13db8304f7a1523fd0f0b7bb Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Fri, 1 Sep 2023 15:52:57 +0200
Subject: [PATCH 0508/1141] PCI/AER: export pci_aer_available

pci_aer_available() can be used in stm32_pcie.c built as a module.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Ic95e9a3bcc9f9c4718467a019b767da283175b5d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/325458
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 drivers/pci/pcie/aer.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/pcie/aer.c b/drivers/pci/pcie/aer.c
index 5426f450ce91..825a4d49ca9e 100644
--- a/drivers/pci/pcie/aer.c
+++ b/drivers/pci/pcie/aer.c
@@ -118,6 +118,7 @@ bool pci_aer_available(void)
 {
 	return !pcie_aer_disable && pci_msi_enabled();
 }
+EXPORT_SYMBOL_GPL(pci_aer_available);
 
 #ifdef CONFIG_PCIE_ECRC
 
-- 
2.39.2

